site/viamguides/src/sticker-wizard/sticker-wizard.md

```diff
@@ -0,0 +1,191 @@
+categories: Getting-Started, Developer
+environments: web
+status: Unpublished
+feedback link: https://github.com/viam-labs/viamcodelabs/issues
+tags: Getting Started, Developer
+
+# Make your own sticker vending machine
+
+<!-- ------------------------ -->
+
+## Overview
+
+Duration: 1
+
+Sticker wizard is a sticker vending machine with a twist: you don't get to choose which sticker you get. Instead, you'll scan the QR code, take the "which robot are you?" personality quiz, and then receive the sticker you deserve (along with a free personality analysis). This tutorial will walk you through making your own vending machine from scratch, along with a web application that allows you to operate your machine from any device.
+
+![sticker wizard](assets/machine.jpeg)
+
+TODO - publish walkthrough video to youtube
+<!-- <video src="assets/sticker-wizard-walkthrough.mov" width="320" height="240" controls></video> -->
+
+### Prerequisites
+
+- Sign up for a free Viam account, and then [sign in](https://app.viam.com).
+- Hardware and supplies requirements
+  - Raspberry Pi
+  - x3 black acrylic sheets (add dimensions, link)
+  - x3 packs of [craft wire](https://www.amazon.com/TecUnite-Aluminum-Bendable-Skeleton-Thickness/dp/B08GCPWVSZ/)
+  - x9 [360° micro servo motors (we used MG90S)](https://www.amazon.com/Compatible-Raspberry-Project-Helicopter-Airplane/dp/B0925TDT2D?th=1)
+  - x1 [16 channel 12-Bit PWM servo motor driver (PCA9685)](https://www.amazon.com/HiLetgo-PCA9685-Channel-12-Bit-Arduino/dp/B07BRS249H)
+  - x2 5V power supply
+- Software
+  - Install the Typescript SDK on your computer.
+
+### What You’ll Learn
+
+- how to be sad
+
+### What You’ll Need
+
+- A computer running Mac, Windows, or Linux
+- [VS Code](https://code.visualstudio.com/download) installed, or your preferred code editor
+- [Python3](https://www.python.org/downloads/) installed
+- Tools
+  - A computer
+  - Laser cutter
+  - 3D printer
+  - Glue / screwdriver
+  - Wire cutter
+
+### What You’ll Build
+
+- A vending machine controlled by a web application.
+
+<!-- ------------------------ -->
+
+## Set up the electronics
+
+### Set up your Raspberry Pi
+Follow the Viam [documentation](https://docs.viam.com/installation/prepare/rpi-setup/) to set up your Raspberry Pi with the Viam RDK.
+
+### Set up the dispensing mechanisms
+1. Wind up your craft wire into 9 coils of equal size 
+    - Our coils are about 20 loops long and have a diameter just under 2". We recommend using a PVC pipe or otherwise cylindrical object as a guide!
+
+<div>
+  <img width="500" src="assets/coil.jpeg">
+</div>
+
+2. Attach the coils to the motors
+    - We simply wrapped the wire around the double servo arms that come with the motors.
+3. Wire up the motors to the motor board
+4. Wire up the motor board to a 5V power supply
+5. Wire up the motor board to your Raspberry Pi
+    - Follow these [instructions](https://learn.adafruit.com/adafruit-16-channel-servo-driver-with-raspberry-pi/hooking-it-up)
```

Comment:
I can create a [Fritzing](https://fritzing.org/) diagram to embed in this codelab to help visualize the wiring.

Reply:
that would be super helpful!

Reply:
![sticker-wizard-diagram](https://github.com/user-attachments/assets/22576cb9-a5b4-4e8c-b74c-b251a32c9053)

This gets a little messy but matches the implementation fairly well. Feel free to add it when you have some time.